gui/builtinStatsViews/targetingMiscViewFull.py

```diff
@@ -197,7 +197,12 @@ def refreshPanel(self, fit):
                     elif labelName == "labelFullSigRadius":
                         label.SetToolTip(wx.ToolTip("Probe Size: %.3f" % (fit.probeSize or 0)))
                     elif labelName == "labelFullWarpSpeed":
-                        label.SetToolTip(wx.ToolTip("Max Warp Distance: %.1f AU" % fit.maxWarpDistance))
+                        maxWarpDistance = "Max Warp Distance: %.1f AU" % fit.maxWarpDistance
+                        if fit.ship.getModifiedItemAttr("warpScrambleStatus"):
+                            warpScrambleStatus = "Warp Core Strength: %.1f" % abs(fit.ship.getModifiedItemAttr("warpScrambleStatus"))
```

Comment:
Is there a reason for the abs() function here? I would think that these would always be positive, and if we were to somehow get a negative here that could lead to incorrect data.

Reply:
Warp core stabs and hull bonuses appear to apply a negative modifier, so having a +2 strength bonus comes out as -2. I thought it would be confusing to people to see -2 as all the in game descriptions imply +2.

Like I mentioned the projected warp disruption does not seem to amend the warp core strength so there isn't currently a way to get erroneous results yet, but if I expand it to include that it would need to be looked at.

Reply:
Ah, yeah, because `warpScrambleStatus` would indicate something has you scrammed, so the stregnth is the inverse. I would recommend changing this then to `fit.ship.getModifiedItemAttr("warpScrambleStatus") * -1` to flip the sign.

And yes, indeed, I've never implemented scram/disruption modifiers for these since we never had a UI need for it. But looking at the effects shows that they do indeed modify the attribute (adds positives). Its up to you if you want this merged in now after the change above or to wait untilt he other effects are fixed up first :)

Reply:
Let's sit on this for now and I'll push a patch set this week once I've read into how projections work a bit.

Reply:
Rightio, just remove the in progress label when you're ready for merging :) (that is, if you have access to removing it. I can never remember)